ports/esp32s2/boards/lilygo_ttgo_xy_esp32_s_woor/mpconfigboard.mk

```diff
@@ -0,0 +1,22 @@
+USB_VID = 0x239A
+USB_PID = 0x80A8
```

Comment:
Please don't reuse the USB PID. https://learn.adafruit.com/how-to-add-a-new-board-to-circuitpython/customizing-the-board-files#mpconfigboard-dot-mk-3033646-25

Reply:
Sorry about that, was under the impression that this would be on the level of the WROOM module. Are VID/PID normally supplied by the board manufacturer?

Reply:
Yes, it's best practice for them to vary per board so that hosts can identify the board based on it. At Adafruit we go even further and have three per board: one for the bootloader, one for arduino and one for circuitpython. Some OSes cache the USB descriptor and/or driver based on the PID so fewer issues occur when separating the three modes.